index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    
+    
+</head>
+<body>
+  <header>
+    <nav>
+      <div class="topnav">
+
+     <h1 class="logo">Grace</h1>
+
+       <div class ='hambuger'>
+        <div class ="hambuger-bar"></div>
+        <div class ="hambuger-bar"></div>
+        <div class ="hambuger-bar"></div>
+
+       </div>
+
+      </div>
+    </nav>
+  </header>
+  <main>
+<section class ="container ">
+  <div class ="first-section" >
+    <div>
+    <h1 class ="title">
+      I'm Grace <br> Glad to see you!
+    </h1>
+    <p class="intro">
+      I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+    </p>
+    <h3 class = "getintouch">
+      LET'S CONNECT
+    </h3>
+    </div>
+    <div>
+      <a class = "socialmedia" href="#"><img src="./images/twitter.png" alt="twitter"></a>
+      <a class = "socialmedia"href="#"><img src="./images/linkedin.png" alt="linkedin"></a>
+      <a class = "socialmedia"href="#"><img src="./images/medium.png" alt="medium"></a>
+      <a class = "socialmedia" href="#"><img src="./images/github.png" alt="github"></a>
+      <a class = "socialmedia"href="#"><img src="./images/anglelist.png"alt="anglelist"></a>
+
+    </div>
```

Comment:
- [ ] Kindly I recommend you to use` ul `tag to group them, and `li` per icon; each icon has a link `a` tag for the social media icons , Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using `href`.

Reply:
But the requirements are that we should use the images from figma